src/Core/Scheduler/Providers/ThreeDTiles_Provider.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Created On: 2016-11-8
+ * Class: ThreeDTiles_Provider
+ * Description:
+ */
+import * as THREE from 'three';
+import Provider from './Provider';
+import CacheRessource from './CacheRessource';
+import B3dmLoader from '../../../Renderer/ThreeExtended/B3dmLoader';
+import Fetcher from './Fetcher';
+import FeatureMaterial from '../../../Renderer/FeatureMaterial';
+
+
+export function ThreeDTilesIndex(tileset) {
```

Comment:
I agree with ThreeDTiles being an awkward name. I can't think of better names than yours, so I'll use them next time I refactor.